pkg/describe/describe_affected_test.go

```diff
@@ -77,3 +77,36 @@ func TestDescribeAffectedWithTargetRepoPath(t *testing.T) {
 
 	t.Log(fmt.Sprintf("\nAffected components and stacks:\n%v", affectedYaml))
 }
+
+func TestDescribeAffectedFiltersDisabledComponents(t *testing.T) {
+	configAndStacksInfo := schema.ConfigAndStacksInfo{}
+
+	atmosConfig, err := cfg.InitCliConfig(configAndStacksInfo, true)
+	assert.Nil(t, err)
+
+	// Set the base path to the test fixtures directory
+	atmosConfig.BasePath = "./tests/fixtures/scenarios/complete"
+
+	// Point to the same local repository
+	repoPath := "../../"
+
+	affected, _, _, _, err := e.ExecuteDescribeAffectedWithTargetRepoPath(
+		atmosConfig,
+		repoPath,
+		true,
+		true,
+		true,
+		"tenant1-ue2-test-1", // Use a specific stack that has disabled components
+	)
+	assert.Nil(t, err)
+
+	// Convert to YAML for logging
+	affectedYaml, err := u.ConvertToYAML(affected)
+	assert.Nil(t, err)
+	t.Log(fmt.Sprintf("\nAffected components and stacks:\n%v", affectedYaml))
```

Comment:
t.Log supports format specifiers, so no need to use fmt.Sprintf

Reply:
removed test as it was not valid